scheduler/plugin/plugins_test.go

```diff
@@ -257,6 +258,63 @@ func Test_NewPluginConfig(t *testing.T) {
 			}(),
 			wantErr: false,
 		},
+		{
+			name: "success with plugin config on Args.Raw ",
+			pc: []v1beta2.PluginConfig{
+				{
+					Name: "InterPodAffinity",
+					Args: runtime.RawExtension{
+						Raw: func() []byte {
+							cfg := v1beta2.InterPodAffinityArgs{
+								TypeMeta: metav1.TypeMeta{
+									Kind:       "InterPodAffinityArgs",
+									APIVersion: "kubescheduler.config.k8s.io/v1beta2",
+								},
+								HardPodAffinityWeight: &hardPodAffinityWeight,
+							}
+							d, _ := json.Marshal(cfg)
+							return d
+						}(),
+					},
+				},
+			},
+			want: func() []v1beta2.PluginConfig {
+				pc := defaultPluginConfig()
+				for i := range pc {
+					if pc[i].Name == "InterPodAffinity" {
+						pc[i] = v1beta2.PluginConfig{
+							Name: "InterPodAffinity",
+							Args: runtime.RawExtension{
+								Object: &v1beta2.InterPodAffinityArgs{
+									TypeMeta: metav1.TypeMeta{
+										Kind:       "InterPodAffinityArgs",
+										APIVersion: "kubescheduler.config.k8s.io/v1beta2",
+									},
+									HardPodAffinityWeight: &hardPodAffinityWeight,
+								},
+							},
+						}
+					}
+					if pc[i].Name == "InterPodAffinityForSimulator" {
+						pc[i] = v1beta2.PluginConfig{
+							Name: "InterPodAffinityForSimulator",
+							Args: runtime.RawExtension{
+								Object: &v1beta2.InterPodAffinityArgs{
+									TypeMeta: metav1.TypeMeta{
+										Kind:       "InterPodAffinityArgs",
+										APIVersion: "kubescheduler.config.k8s.io/v1beta2",
+									},
+									HardPodAffinityWeight: &hardPodAffinityWeight,
+								},
+							},
+						}
+					}
+				}
+
+				return pc
+			}(),
+			wantErr: false,
```

Comment:
Would you like to add a test case to verify
> 			// If Args data exists in both PluginConfig.Args.Raw and PluginConfig.Args.Object, PluginConfig.Args.Object takes precedence.

Reply:
Sure. 👍